python/res/enkf/config/enkf_config_node.py

```diff
@@ -114,6 +194,11 @@ def createFieldConfigNode(cls, key, grid, trans_table = None, forward_init = Fal
         """
         return cls._alloc_field_node(key, grid, trans_table, forward_init)
 
+    @classmethod
+    def get_real_path(cls, path):
```

Comment:
Is probably not needed as a class method. Since it is only used one place, os.path.realpath can be used directly.

Reply:
ok